Models/reviews.models.js

```diff
@@ -30,8 +30,32 @@ exports.updateReview = (review_id, alterVotes) => {
     })
 }
 
-exports.fetchReviews = () => {
-    return db.query("SELECT reviews.review_id, reviews.title, reviews.owner, reviews.review_img_url, reviews.category, reviews.created_at, reviews.votes, COUNT(comments.review_id)::INT AS comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY created_at  DESC")
+exports.fetchReviews = (sort_by = "created_at", order = "DESC", category) => {
+    let validSortBy = ["review_id", "created_at", "votes", "comment_count"]
+    let validOrder = ["ASC", "DESC"]
+    let validCategory = ["euro game", "dexterity", "social deduction"]
```

Comment:
Change these to const's, no need for them to be let's

Reply:
Also would like you to think about the fact that this list is static, if we added categories in future, we would have to come and update this code. It may be better to fetch the categories from the categories table to ensure that we are checking all categories. (You dont need to do this for this ticket but is needed for a later ticket)

Reply:
Also you are missing a category here, check your test-data and then include a test for if the category exists but there are currently no reviews in it